orca-webhook/orca-webhook.gradle

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017 Schibsted ASA.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+dependencies {
+  compile project(':orca-core')
+  compile spinnaker.dependency('kork')
+  compile spinnaker.dependency('bootAutoConfigure')
+  compile group: 'com.jayway.jsonpath', name: 'json-path', version: '2.2.+'
```

Comment:
We might want to get the same declaration going on here, as we already have a dep on jsonpath here: https://github.com/spinnaker/orca/blob/master/orca-pipelinetemplate/orca-pipelinetemplate.gradle#L12

Might be worthwhile getting this into https://github.com/spinnaker/spinnaker-dependencies so there's only one place the dep version is defined.